src/App.js

```diff
@@ -1,23 +1,38 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState, useEffect } from 'react'
+import beersService from './services/beers'
+import BeersList from "./components/BeersList/BeersList";
+import Home from "./components/Home/Home";
+
+import { Navigate, Route, Routes } from 'react-router-dom';
+import SingleBeer from './components/SingleBeer/SingleBeer';
+import RandomBeer from './components/RandomBeer/RandomBeer';
+import NewBeer from './components/NewBeer/NewBeer';
 
 function App() {
+  const [beers, setBeers] = useState([])
+
+  useEffect(() => {
+    beersService.list()
+      .then((beers) => {
+        setBeers(beers)
+      })
+      .catch(error => console.error(error))
+  }, [])
+
+  const addBeer = (beer) => {
+    setBeers([beer, ...beers])
+  }
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div >
```

Comment:
```suggestion
    <>
```

Yo aqui te recomiendo usar mejor los fragment o directamente el Routes, pero si te queda mas claro un fragment y asi te ahorras un div que ni pincha ni corta ahi